README.rst

```diff
@@ -735,14 +735,21 @@ The filter can have the following parameters:
   This parameter specifies which character will be used to identify the
   Logstash section.
 
+- ``backslash_escaping_ignore_string='@@@'``
```

Comment:
Please format it like this (empty line  behind the option name, example right behind the text, minor text corrections):

```rst
- ``backslash_escaping_ignore_string='@@@'``

  This parameter defines a sets of characters than can be prepended to a string
  to prevent backslahes from being escaped in the resulting configuration (e.g.
  ``"@@@sshd(?:\[%{POSINT:[system][auth][pid]}\])?:"`` will turn to
  ``"sshd(?:\[%{POSINT:[system][auth][pid]}\])?:"`` instead of to
  ``"sshd(?:\\[%{POSINT:[system][auth][pid]}\\])?:"``).
```

Reply:
Updated pull request with your proposed changes.